### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by invictusro** [scivision/PyLivestream#80]
Hi, is there any way I can stream to another service than the ones presented if I have stream key and stream URL ? 
Basically to stream to a RTMP server, but not for the ones present there ?

**Comment by tabanger** [comment]
Yes, you can stream to any RTMP server; just add your own server configuration in the pylivestream.ini file.  Use the other sites as an example.  For example, you could add this section to the ini file:

[myserver]
server: rtmp://your.server.address
key: ~/yourkey.key
